coordinator/migrations/2023-09-04-090152_add_order_state_and_matches/up.sql

```diff
@@ -10,3 +10,22 @@ ALTER TABLE "orders" ADD COLUMN "order_state" "OrderState_Type" NOT NULL DEFAULT
 
 UPDATE orders SET order_state = 'Taken' WHERE taken = true;
 UPDATE orders SET order_state = 'Failed' WHERE taken = false;
+
+CREATE TYPE "MatchState_Type" AS ENUM (
```

Comment:
Changes in this file should be in an separate migration file or all in the same file.
If we end up deploying the commit which creates this file before this commit here, we will not re-run the migration and will get an error.

Reply:
I've split up the migration scripts into dedicated folders per commit.